modules/op-release-notes-1-19.adoc

```diff
@@ -450,25 +453,17 @@ platforms:
 *** Global. This option applies to all namespaces except those prefixed with `kube-` and `openshift-`. 
 *** Namespace. This option applies to all resources in a specific namespace
 +
---
-:FeatureName: The Pruner component
-include::snippets/technology-preview.adoc[]
 [NOTE]
 ====
 In this release, only `Global` and `Namespace` configurations are available.
 ====
---
 
-* With this update, the event-based pruner can now be enabled or disabled through the `TektonConfig` CR by setting the `spec.tektonpruner.disabled` parameter to `false`. Fine-grained configuration is not yet available through the `TektonConfig` CR and should be managed using config maps.
+* With this update, the event-based pruner can now be enabled or disabled through the `TektonConfig` CR by setting the `spec.tektonpruner.disabled` parameter to `false` or `true`. Fine-grained configuration is not yet available through the `TektonConfig` CR and should be managed using config maps.
```

Comment:
Suggestion: Can we change to this:

* With this update, you can enable or disable the event-based pruner in the `TektonConfig` CR by setting `spec.tektonpruner.disabled` to `false` or `true`. Fine-grained configuration is not yet supported in the `TektonConfig` CR and must be managed through config maps.